### PR TITLE
docs: update tutorials/faq structure

### PR DESCRIPTION
**Pull request by ndom91**
## Reasoning 💡

- clean up css
  - there was a lot of totally unused css, and the navbar bits were growing large enough to warrant their own file imo
- improve tutorial page readability + organisation
- upgrade docusaurus to latest beta release

## Checklist 🧢

- [x] Documentation
- [ ] Tests
- [x] Ready to be merged

## Screenshot:

New tutorial page, reorganised a bit and with icons to denote youtube video / external tutorial:
image

~Also I took a stab at the FAQ page because it was super hard to read imo. With html `<detail>` and `<summary>` elements, we can very easily convert it to a full accordion style faq page. What do you guys think about this (first section only as demo - didnt 
 commit yet, wanted your feedback first):~

nextauth_faqAccordion

~With this approach, the `h3` headers which contain the faq questions, are no longer automatically linked as IDs with which you can navigate directly to hashes in the URL. Is that a large loss? What do you think? If we can live with that and you guys also find this much more readable / navigable compared to the current incarnation, I can finish off the rest of the FAQ questions with summary and detail etc and add it to this PR as well :+1:~

--- 
**EDIT**: 

Yeah so the FAQ page looks muchh better imo, way easier to navigate, find things at a glance, etc. :+1: 

image
image

### CODE REVIEW

**Comment by balazsorban44** [comment]
How does summary/detail affect Algolia and inline search in general? I think that would be a big enough loss that I wouldn't prefer this change. Most of those would read the FAQ might need the help of a search probably.

**Comment by ndom91** [comment]
Yeah I think algolia search wouldn't be affected since it just scrapes the html 1x daily. But I'd have to do some more research to be sure. 

I agree that's too much of a loss if that doesn't work. I'll look into it some more and come back 👍

**Comment by ndom91** [comment]
Okay so it looks like the algolia docsearch will still work as before. When parsing the rendered HTML, there are still `h2` and `p` tags inside the summary/details. The summary/details just confuses docusaurus at build time and so doesn't generate the ID's.

> It automatically follows every internal link to make sure we are not missing any content, and uses the semantics of your HTML structure to construct its records. This means that h1,h2, etc., (selectors) titles are used as hierarchy, and each p is used as a 
 potential result.
> Source: https://docsearch.algolia.com/docs/scraper/

File: www/docs/tutorials.md [hunk 1]
```diff
@@ -3,94 +3,96 @@ id: tutorials
...
 
-### [LDAP Authentication](tutorials/ldap-auth-example)
+- Learn how to add Twitter authentication and login to a Next.js app both clientside and serverside with NextAuth.js.
 
-How to use the Credentials Provider to authenticate against an LDAP database.
+#### [NextJS Authentication Crash Course with NextAuth.js](https://youtu.be/o_wZIVmWteQ) <svg role="img" viewBox="0 0 24 24" height="24" width="24" style={{ marginLeft: '5px', marginBottom:'-6px'}} xmlns="http://www.w3.org/2000/svg"><title>YouTube</title><path fill="#ff0000" d="M23.498 6.186a3.016 3.016 0 0 0-2.122-2.136C19.505 3.545 12 3.545 12 3.545s-7.505 0-9.377.505A3.017 3.017 0 0 0 .502 6.186C0 8.07 0 12 0 12s0 3.93.502 5.814a3.016 3.016 0 0 0 2.122 2.136c1.871.505 9.376.505 9.376.505s7.505 0 9.377-.505a3.015 3.015 0 0 0 2.122-2.136C24 15.93 24 12 24 12s0-3.93-.502-5.814zM9.545 15.568V8.432L15.818 12l-6.273 3.568z"/></svg>
 
-This approach can be used to authenticate existing user accounts against any backend.
+- This tutorial dives in to the ins and outs of NextAuth including email, Github, Twitter and integrating with Auth0 in under hour.
```
Comment (lluia) [inline www/docs/tutorials.md:42]:
```suggestion
- This tutorial dives into the ins and outs of NextAuth, including using the Email, Github, Twitter and Auth0 providers in under an hour.
```

File: www/docs/tutorials.md [hunk 2]
```diff
@@ -3,94 +3,96 @@ id: tutorials
...
 
-A tutorial by Arunoda Susirpiala. Checkout [GetStarted](https://getstarted.sh/) for more examples.
+#### [Fullstack Authentication Example with Next.js and NextAuth.js](https://github.com/prisma/prisma-examples/tree/latest/typescript/rest-nextjs-api-routes-auth) <svg xmlns="http://www.w3.org/2000/svg" style={{ marginLeft: '5px', marginBottom:'-6px'}} height="20" width="20" fill="none" viewBox="0 0 24 24" stroke="currentColor"><title>External</title> <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M10 6H6a2 2 0 00-2 2v10a2 2 0 002 2h10a2 2 0 002-2v-4M14 4h6m0 0v6m0-6L10 14" /> </svg>
 
-### [Add auth support to a Next.js app with a custom backend](https://arunoda.me/blog/add-auth-support-to-a-next-js-app-with-a-custom-backend)
+- This example shows how to implement a fullstack app in TypeScript with Next.js using React (frontend), Next.js API routes and Prisma Client (backend). It also demonstrates how to implement authentication using NextAuth.js. By Nikolas Burk at Prisma.
```
Comment (lluia) [inline www/docs/tutorials.md:60]:
```suggestion
- This example shows how to implement a full-stack app in TypeScript with Next.js using Prisma Client as a backend. It also demonstrates how to implement authentication using NextAuth.js. By Nikolas Burk at Prisma.
```
You can only use React in NextJS 😄 , hence I think we could re-phrase this one 💅🏽

File: www/docs/tutorials.md [hunk 3]
```diff
@@ -3,94 +3,96 @@ id: tutorials
...
+- A tutorial by Arunoda Susirpiala.
 
-This example shows how to implement a fullstack app in TypeScript with Next.js using React (frontend), Next.js API routes and Prisma Client (backend). It also demonstrates how to implement authentication using NextAuth.js. By Nikolas Burk at Prisma.
+#### [How to Configure Azure AD B2C Authentication with Next.js](https://benjaminwfox.com/blog/tech/how-to-configure-azure-b2c-with-nextjs) <svg xmlns="http://www.w3.org/2000/svg" style={{ marginLeft: '5px', marginBottom:'-6px'}} height="20" width="20" fill="none" viewBox="0 0 24 24" stroke="currentColor"><title>External</title> <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M10 6H6a2 2 0 00-2 2v10a2 2 0 002 2h10a2 2 0 002-2v-4M14 4h6m0 0v6m0-6L10 14" /> </svg>
 
-### [Adding Authentication to an existing Next.js Application in no time!](https://dev.to/ndom91/adding-authentication-to-an-existing-serverless-next-js-app-in-no-time-with-nextauth-js-192h)
+- Configuring authentication with Azure B2C in Next.js is not a particularly straight forward process. We'll look at how to facilitate this using the NextAuth.js library. By Ben Fox.
```
Comment (lluia) [inline www/docs/tutorials.md:84]:
```suggestion
- Configuring authentication with Azure B2C in Next.js is not a particularly straightforward process. The tutorial shows how to facilitate this using the NextAuth.js, by Ben Fox.
```

File: www/docs/tutorials.md [hunk 4]
```diff
@@ -3,94 +3,96 @@ id: tutorials
...
 
-This `dev.to` tutorial walks one through adding NextAuth.js to an existing project. Including setting up the OAuth client id and secret, adding the API routes for authentication, protecting pages and API routes behind that authentication, etc.
+#### [Adding Sign in With Apple Next JS](https://thesiddd.com/blog/apple-auth) <svg xmlns="http://www.w3.org/2000/svg" style={{ marginLeft: '5px', marginBottom:'-6px'}} height="20" width="20" fill="none" viewBox="0 0 24 24" stroke="currentColor"><title>External</title> <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M10 6H6a2 2 0 00-2 2v10a2 2 0 002 2h10a2 2 0 002-2v-4M14 4h6m0 0v6m0-6L10 14" /> </svg>
```
Comment (lluia) [inline www/docs/tutorials.md:86]:
```suggestion
#### [Sign in with Apple in NextJS](https://thesiddd.com/blog/apple-auth) <svg xmlns="http://www.w3.org/2000/svg" style={{ marginLeft: '5px', marginBottom:'-6px'}} height="20" width="20" fill="none" viewBox="0 0 24 24" stroke="currentColor"><title>External</title> <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M10 6H6a2 2 0 00-2 2v10a2 2 0 002 2h10a2 2 0 002-2v-4M14 4h6m0 0v6m0-6L10 14" /> </svg>
```

File: www/docs/tutorials.md [hunk 5]
```diff
@@ -3,94 +3,96 @@ id: tutorials
...
 
-### [Introduction to NextAuth.js](https://www.youtube.com/watch?v=npZsJxWntJM)
+- This tutorial walks step by step on how to get Sign In with Apple working (both locally and on a deployed website) using NextAuth.js.
```
Comment (lluia) [inline www/docs/tutorials.md:88]:
```suggestion
- This tutorial walks step by step on how to get sign in with Apple working (both locally and on a deployed website) using NextAuth.js.
```

File: www/src/css/index.css [hunk 6]
```diff
@@ -112,34 +88,17 @@ a:hover,
...
-  justify-content: center;
-  padding: 0rem 0 2rem 0;
-  opacity: 0.6;
-  font-style: italic;
-  margin: 0 auto;
-  font-size: 1.2rem;
-  text-align: center;
+html[data-theme="dark"] {
+  --ifm-background-color: #1f201c;
 }
```
Comment (lluia) [inline www/src/css/index.css:102]: I would move variable declarations and their overrides at the top of this `index.css` file, so they're easy to discover 👍🏽

**Review by 0ubbe — APPROVED** [review]
Super 💯 💚 , minor nitpick and suggestions 💬

File: www/src/css/index.css (hunk 6 above)
Comment (ndom91) [reply www/src/css/index.css]: Good call :+1:

**Review by 0ubbe — APPROVED** [review]
Lets get this in 💯 💚 💪🏽

**Pull request merged**